client/consensus/grandpa/utils.go

```diff
@@ -0,0 +1,29 @@
+// Copyright 2023 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package grandpa
+
+import (
+	"fmt"
+)
+
+func bytesToHash(b []byte) Hash {
```

Comment:
I don't think you need this.  You only use it for tests, to generate something that satisfies `comparable`.  You can just use the strings in the tests.

Reply:
Really good call on this. Was able to remove whole file and clean up the code